pegasus/sites.v3/hourofcode.com/views/language_dropdown.haml

```diff
@@ -1,5 +1,6 @@
 -path = request.path_info
 -path = path[@language.length + 1..-1] if path[0..@language.length] == "/#{@language}"
-%select{onchange:"window.location.href = '/#{@country}/' + this.value + '#{path != '/' ? path : ''}';"}
+%label{class: 'hidden-label', for: 'language-dropdown-select'} Language
```

Comment:
I'm not super familiar with HAML - is it possible to have 'Language' be translatable?

Reply:
oh yeah, it can, I spaced on that. Will update!